### PR TITLE
Configure new docs deploy

### PR DESCRIPTION
**Pull request by elmarques**
# What

This PR configures the deployment of the new documentation.

# Why

For Astro documentation to be publicly available.

# How

* Isolates and organizes all documentation files into the `./docs` folder
* Configures the static Storybook build
* Updates the deploy settings on Netlify

# Refs

* [Clubhouse card](https://app.clubhouse.io/magnetis/story/35933/configurar-storybook-e-netlify-para-deploy-da-nova-documentação) [ch35933]

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -65,9 +65,8 @@ lib/
 .jest-cache
 .DS_Store
 
-# gatsby files
-packages/docs/.cache/
-packages/docs/public
-
```
Comment (AgtLucas) [inline .gitignore:71]: Ops, I forgot these ones. 🙈

File: package.json [hunk 2]
```diff
@@ -56,6 +57,7 @@
     "eslint-plugin-prettier": "^3.1.0",
     "eslint-plugin-react": "7.x",
     "eslint-plugin-react-hooks": "1.5.0",
+    "file-loader": "^6.0.0",
```
Comment (AgtLucas) [inline package.json:60]: Do we need this?

Comment (elmarques) [reply package.json:60]: Yes. To read locally some PNGs for home. 😿 

I tried to use the Astro CSS remote PNGs, but they have a dynamic hashed name.

Comment (AgtLucas) [reply package.json:60]: I see :D

**Pull request merged**